kubescape/operator
# feat: schedule CVE scans for new SBOMs

## Overview


This PR adds support for scheduling CVE scans for newly detected SBOMs, as required by the relevancy feature.



## How to Test

To test, install the `relevancy` Kubescape Helm chart with the `kubescapeStorage.enabled=true` and observe the behavior of the Operator.





## Checklist before requesting a review

put an [x] in the box to get it checked
